sklearn/cluster/tests/test_optics.py

```diff
@@ -0,0 +1,137 @@
+from sklearn.datasets.samples_generator import make_blobs
+from sklearn.cluster.optics import OPTICS
+from sklearn.utils.testing import assert_equal, assert_greater_equal
+from .common import generate_clustered_data
+import numpy as np
+
+
+def test_optics():
+    '''
```

Comment:
Please don't use docstrings for the test functions as it makes debugging a bit harder in `nosetests -v`